_api-reference/index-apis/clone.md

```diff
@@ -55,6 +55,8 @@ Parameter | Type | Description
 wait_for_active_shards | String | The number of active shards that must be available before OpenSearch processes the request. Default is 1 (only the primary shard). Set to all or a positive integer. Values greater than 1 require replicas. For example, if you specify a value of 3, the index must have two replicas distributed across two additional nodes for the operation to succeed.
 master_timeout | Time | How long to wait for a connection to the master node. Default is `30s`.
 timeout | Time | How long to wait for the request to return. Default is `30s`.
+wait_for_completion | Boolean | When set to `false`, the request will return a task immediately and the operation will run in background. Default is `true`.
```

Comment:
I assume it's the task **ID** that's returned? And then you use that task ID in the tasks API to check on the task? If so, add this information and link to the tasks API.
```suggestion
wait_for_completion | Boolean | When set to `false`, the request will return immediately instead of waiting for the operation to finish. Default is `true`.
```

Reply:
yes, a task ID is returned. I'll get this updated.